api/app.js

```diff
@@ -27,7 +27,7 @@ const instructorRouter = require('./instructor/instructorRouter');
 const childrenRouter = require('./children/childrenRouter');
 const inboxRouter = require('./inbox/inboxRouter');
 const programTypesRouter = require('./programs/programTypesRouter');
-const classInstancesRouter = require('./classInstances/classInstancesRouter');
+const coursesRouter = require('./courses/coursesRouter');
```

Comment:
Great. "courses" is much more comprehensible than "classInstance" (typ.)

Reply:
Definitely! Glad to see us implement this change.